scholarly/_navigator.py

```diff
@@ -149,12 +136,44 @@ def _use_proxy(self, http: str, https: str):
         """ Routes scholarly through a proxy (e.g. tor).
             Requires pysocks
             Proxy must be running."""
-        self.logger.info("Enabling proxies: http=%r https=%r", http, https)
+        self.logger.info("Enabling proxies: http=%r https=%r", http, https)        
+        self._http_proxy = http
+        self._https_proxy = https
+
+    def _use_tor(self, tor_cmd=None, tor_sock_port=_SCHOLAR_TOR_SOCK_PORT, tor_control_port=_SCHOLAR_TOR_CONTROL_PORT):
+        '''
+        Starts a Tor client running in a schoar-specific port, together with a 
+        scholar-specific control port.
+        '''
+        self.logger.info("Starting tor as the proxy")
+        
+        if tor_cmd is None:
```

Comment:
This not necessarily works. There are several comments on stem about incompatibilities with windows. Also tor .exe can be located elsewhere.